app/User.php

```diff
@@ -48,4 +48,9 @@ public function getJWTCustomClaims()
     {
         return [];
     }
+
+    public function channels()
+    {
+        return $this->belongsTo('App\Channels');
```

Comment:
```suggestion
        return $this->hasOne(Channels::class);
```

Usar a sugestão acima, incluindo `use App\Channels` no começo da classe. Em qualquer lugar que o nome de uma classe for necessária, usar `Nome::class` ao invés de uma string.